plugins/services/src/js/filters/ServiceAttribHealthFilter.js

```diff
@@ -0,0 +1,43 @@
+import DSLFilterTypes from '../../../../../src/js/constants/DSLFilterTypes';
+import DSLFilter from '../../../../../src/js/structs/DSLFilter';
+import HealthStatus from '../constants/HealthStatus';
+
+const LABEL_TO_HEALTH = {
+  healthy: HealthStatus.HEALTHY,
+  idle: HealthStatus.IDLE,
+  na: HealthStatus.NA,
+  unhealthy: HealthStatus.UNHEALTHY
+};
+
+/**
+ * This filter handles the `health:status` for services using `getHealth`
+ */
+class SearviceAttribHealthFilter extends DSLFilter {
+
+  /**
+   * Handle all `health:XXXX` attrib filters that we can handle.
+   *
+   * @override
+   */
+  filterCanHandle(filterType, filterArguments) {
```

Comment:
Yes, prefixing is important since these functions are implemented on an abstract base class, meaning that a user might decide to implement additional functions and I want to avoid naming collisions.

Take for example React and `componentXXXX` -prefixed functions